desktop/apps/auctions2/components/Layout.js

```diff
@@ -1,27 +1,40 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import { connect } from 'react-redux'
+import AuctionsBlock from 'desktop/apps/auctions2/components/auctions_block/AuctionsBlock'
 
-function Layout ({ isMobile }) {
+function Layout ({ isMobile, auctions }) {
   return (
     <div>
       {isMobile
         ? <div>
             Hello Mobile!
           </div>
         : <div>
-            Hello Desktop!
+          {auctions.live.length ?
```

Comment:
I feel so-so about re-using `live` here, since a similar-looking query in gravity would return something different. (`live` there means `open for bidding`). I know it's more verbose, but can we rename this scoping to `liveIntegration`? 😄